docs/front-end/grid.md

```diff
@@ -4,43 +4,40 @@ The grid setup can be found in `sass/components/_grid.scss`.
 
 The site uses a grid that changes with the breakpoints of the site. Above the large breakpoint `(min-width: 1023px)` the grid is split into 12 columns and below the large breakpoint the grid is split into 4 columns.
```

Comment:
While you're updating this, could you amend this as it is actually 14 columns, with 12 equal then one at either end with a fixed width.

Reply:
Good shout - updated in https://github.com/torchbox/torchbox.com/pull/69/commits/c90331be69dc960eb7f805912cfe721a5e09a7f8